pkg/types/defaults.go

```diff
@@ -10,9 +10,10 @@ const (
 
 var (
 	NcpSections      = []string{"DEFAULT", "ha", "k8s", "coe", "nsx_v3", "vc"}
-	AgentSections    = []string{"DEFAULT", "k8s", "coe", "nsx_node_agent", "nsx_kube_proxy"}
+	AgentSections    = []string{"DEFAULT", "nsx_node_agent", "nsx_kube_proxy"}
```

Comment:
We should not remove k8s right, you need to compare apiserver_host_ip in AgentRestartKeys

Reply:
Another way I'm thinking is to only check options instead of checking sections

Reply:
In current implementation, we need to remove `k8s` and `coe` section, and I added key like `apiserver_host_ip` in `AgentRestartKeys`. This means that only certain key like `apiserver_host_ip` change will trigger operator to restart node-agent.